samples/wifi/radio_test/sample_description.rst

```diff
@@ -125,7 +125,7 @@ Testing
               wlan_ant_switch_ctrl = 0
 
 
-         * To run a continuous Orthogonal frequency-division multiplexing (OFDM) TX traffic sequence with the following configuration:
+         * To run a continuous Orthogonal frequency-division multiplexing (OFDM) TX traffic sequence use the following configuration:
```

Comment:
I think "with" is the correct word here. "use" gives a wrong meaning - the tests have to be run "with" the below parameters. This comment is relevant for all such changed